deptofdefense/move.mil
# Moving Guide icons, mobile layout

## Checklist

I have…


## Summary of Changes

This pull request...

- Adds the missing image assets to the moving guide.
- Updates the look of the page to better match the Zeplin design, including
  - Two column list of links at the top of the page
  - Borders around the move type table
- Some minor text updates.
- Adjusts the markup and styles to make the mobile layout behave.
- Makes the Move Types table render identically on Chrome and Firefox.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `moving-page-styles-icons`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.

## Screenshots

### Before
![fireshot capture 48 - moving with the military move mil - http___localhost_3000_moving-guide](https://user-images.githubusercontent.com/26554826/29333237-08cfc242-81d1-11e7-888f-62af7350568c.png)

### After - Desktop
![fireshot capture 49 - moving with the milit_ - http___localhost_3000_moving-guide about-entitlements](https://user-images.githubusercontent.com/26554826/29333280-3056295a-81d1-11e7-8e51-150d1610b9c5.png)

### After - Mobile
![fireshot capture 51 - moving with the milit_ - http___localhost_3000_moving-guide about-entitlements](https://user-images.githubusercontent.com/26554826/29333397-a438fc3a-81d1-11e7-9417-7eca3ebc4ad3.png)
